regression/clear_datastore.py

```diff
@@ -15,13 +15,18 @@
 """Script to populate datastore with regression test data."""
 
 
-# This assumes the command is being run via tox hence the
-# repository root is the current directory.
+from gcloud import datastore
+from gcloud.datastore import _implicit_environ
 from gcloud.datastore.query import Query
-from regression import regression_utils
+from gcloud.datastore.transaction import Transaction
 from six.moves import input
 
 
+datastore._DATASET_ENV_VAR_NAME = 'GCLOUD_TESTS_DATASET_ID'
```

Comment:
`datastore._DATASET_ENV_VAR_NAME` is meant for production code.

When building out the "implicit" feature @silvolu pointed out (and I agree) that we don't want devs to accidentally use a production dataset for running tests, hence the use of a different environment variable.